.secrets.baseline

```diff
@@ -115,7 +115,8 @@
       "pattern": [
         "(?i)dummyapikey",
         "(?i)test-secret",
-        "^x-api-key$"
+        "^x-api-key$",
+        "^1f9d73167e2166b707c6$"
```

Comment:
Similar comment as [here](https://github.com/govuk-one-login/ipv-core-back/pull/2036#discussion_r1647681199) but this secret also gets flagged in a json file